src/Ormolu/Printer/Combinators.hs

```diff
@@ -131,9 +131,17 @@ spansLayout = \case
   [] -> SingleLine
   (x : xs) ->
     if isOneLineSpan (foldr combineSrcSpans x xs)
-      then SingleLine
+      then
+        if isLineTooLong (foldr combineSrcSpans x xs) then
+          MultiLine
+        else
+          SingleLine
       else MultiLine
 
+isLineTooLong :: SrcSpan -> Bool
+isLineTooLong (RealSrcSpan span) = srcSpanEndCol span > 100
+isLineTooLong (UnhelpfulSpan _) = False
+
```

Comment:
@JDemler Wie funktioniert das? Bezieht sich das auf den output oder auf den Input?

Reply:
Wenn die Summe aller Spans die er printen will länger als 100 ist wechselt er in den Multiline Mode.